doc/architecture/uml/rest_service_get_request.wsd

```diff
@@ -0,0 +1,133 @@
+@startuml Get-Request
+participant Plugin
+participant RestService
+participant AsyncHttpClient 
+participant "RestService::m_responseQueue" as responseQ
+participant TcpClient
+entity Server
+
+autoactivate on
+
+-> RestService: start()
+    RestService -> AsyncHttpClient: regOnResponse/Error/Closed
```

Comment:
please write a sentence like you did in the other places, e.g. register callback for ...